lua/notebook-navigator/repls.lua

```diff
@@ -12,7 +12,7 @@ repls.toggleterm = function(start_line, end_line, repl_args)
   local trim_spaces = true
   if repl_args then
     id = repl_args.id or 1
-    trim_spaces = repl_args.trim_spaces or true
+    trim_spaces = (repl_args.trim_spaces == nil) and true or repl_args.trim_spaces
```

Comment:
Good catch!

Looking at it more closely that logic can be further simplified to 
```lua
(repl_args.trim_spaces == nil) or repl_args.trim_spaces
```

because  what we have now is adding parenthesis equivalent to:
```lua
((repl_args.trim_spaces == nil) and true) or repl_args.trim_spaces
```

and `x and true` is just `x`.

Reply:
Indeed! I just used the standard lua "ternary operator" (aka and+or). I fixed it.